### PR TITLE
Review request: Second milestone for react calculator app

### PR DESCRIPTION
**Pull request by chrystalme**
- Created Display, Button, ButtonPanel component
 - Display component has props and props validation for the results
 - Button component has props and props validation for button name
 - ButtonPanel has divs with button group that renders a set of buttons

### CODE REVIEW

File: src/components/app.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import '../App.css';
 function App() {
   return (
 
-    <form action="">
+    <form className="App-header" action="">
```
Comment (whiz25) [inline src/components/app.js:9]: Use React.Fragment short syntax
`<>
...
</>
`

File: src/components/display.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 const Display = (props) => {
   const { result } = props;
   return (
-    <input type="text" value={result} readOnly />
+    <input className="d-flex justify-content-end" type="text" value={result} readOnly />
```
Comment (whiz25) [inline src/components/display.js:7]: The result should have a default value.
`Display.defaulProps = { result: '0' }`

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Awesome job. You have fulfilled most of the requirements for this milestone.
### Highlights 🌵 🌵 🌵 
- `DisplayPanel` and `Button` components well implemented

### Changes 🔕 🔕 🔕 
- Check comments for suggestions


### Martin Afani
#### Slack: whiz25

**Review by bellom — APPROVED** [review]
## Hi @chrystalme  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best**

**Pull request merged**